account_payment_select_account/models/account_payment.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (C) 2016-Today: La Louve (<http://www.cooplalouve.fr/>)
+# Copyright (C) 2019-Today: Druidoo (<https://www.druidoo.io>)
+# @author: La Louve
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html
+
+from odoo import api, fields, models
+
+
+class AccountPayment(models.Model):
+    _inherit = "account.payment"
```

Comment:
You need to inherit `account.payment.register` instead.
Note that, this module asks user to input the account when clicking on the button `Pay` from the invoice.